apps/docs/docusaurus.config.ts

```diff
@@ -0,0 +1,136 @@
+import type { Options as DocGenOption } from "docusaurus-plugin-react-docgen-typescript";
+import type * as Preset from "@docusaurus/preset-classic";
+
+const config: Config = {
+    title: "MAD Platform",
+    tagline: "For all your mobile needs",
+    favicon: "img/favicon.ico",
+
+    // Set the production url of your site here
+    url: "https://equinor.github.io",
+    // Set the /<baseUrl>/ pathname under which your site is served
+    // For GitHub pages deployment, it is often '/<projectName>/'
+    baseUrl: "/mad/",
+
+    // GitHub pages deployment config.
+    // If you aren't using GitHub pages, you don't need these.
+    organizationName: "equinor", // Usually your GitHub org/user name.
+    projectName: "mad", // Usually your repo name.
+
+    onBrokenLinks: "throw",
+    onBrokenMarkdownLinks: "warn",
+
+    // Even if you don't use internationalization, you can use this field to set
+    // useful metadata like html lang. For example, if your site is Chinese, you
+    // may want to replace "en" with "zh-Hans".
+    i18n: {
+        defaultLocale: "en",
+        locales: ["en"],
+    },
+
+    presets: [
+        [
+            "classic",
+            {
+                docs: {
+                    sidebarPath: "./sidebars.ts",
+                    // Please change this to your repo.
+                    // Remove this to remove the "edit this page" links.
+                    editUrl: "https://github.com/equinor/mad/tree/main/apps/docs/shared",
+                },
+                theme: {
+                    customCss: "./src/css/custom.css",
+                },
+            } satisfies Preset.Options,
+        ],
+    ],
+
+    themeConfig: {
+        // Replace with your project's social card
+        image: "img/docusaurus-social-card.jpg",
+        navbar: {
+            title: "MAD",
+            logo: {
+                alt: "the mad logo",
+                src: "img/mad-head-light.svg",
+                srcDark: "img/mad-head-dark.svg",
+            },
+            items: [
+                {
+                    type: "dropdown",
+                    position: "left",
+                    label: "Packages",
+                    items: [
+                        {
+                            type: "docSidebar",
+                            sidebarId: "coreSidebar",
+                            label: "MAD Core",
+                        },
+                        {
+                            type: "docSidebar",
+                            sidebarId: "componentsSidebar",
+                            label: "Components",
+                        },
+                    ],
+                },
+                {
+                    type: "docSidebar",
+                    sidebarId: "tutorialSidebar",
+                    position: "left",
+                    label: "Docusaurus tutorial",
+                },
+                {
+                    href: "https://github.com/facebook/docusaurus",
```

Comment:
Github link in the top right corner should probably link to [https://github.com/equinor/mad](https://github.com/equinor/mad)

Reply:
Fixed